3-platform/order/db/postgres/01-create-orders-table.sql

```diff
@@ -0,0 +1,25 @@
+-- orders table
+
+CREATE TABLE IF NOT EXISTS public.orders (
+    id          varchar     NOT NULL,
+    created_at  timestamp   NOT NULL,
+    total       bigint      NOT NULL,
+    status      varchar     NULL,
```

Comment:
Lets make status not nullable.

Reply:
I'm not sure it's necessary. But I think to set to the 'status' field default value as 'New'.